api/data_explorer/__main__.py

```diff
@@ -237,6 +237,7 @@ def _process_facets(es):
     # Elasticsearch field type, optional UI facet description and Elasticsearch
     # facet.
     app.app.config['FACET_INFO'] = facets
+    app.app.config['EXTRA_FACET_INFO'] = {}
```

Comment:
I think it makes sense to have a proper default value here, but I actually would've thought this would be handled correctly [here](https://github.com/DataBiosphere/data-explorer/blob/4842135d9ac8a6fea67fe1204108c114d9a2844c/api/data_explorer/controllers/facets_controller.py#L28). Where are you seeing the key not found error thrown from?

Reply:
I think [this](https://github.com/DataBiosphere/data-explorer/blob/master/api/data_explorer/controllers/export_url_controller.py#L293) is the line that was throwing (I didn't save the stack trace, but it was in the export controller and that's the first place where the key is accessed). I was hitting the error by:
1. Opening the explorer
2. Checking a few boxes in the pre-curated search facets
3. Clicking the export button

If it matters, I didn't have a `bigquery.json` in my explorer config at the time. I've noticed that some of the configuration code paths only get hit when it's present on deploy.

Reply:
Ah the missing `biquery.json` was probably it, thanks for the explanation!